lib/dart/lib/src/transport/t_buffered_transport.dart

```diff
@@ -22,13 +22,13 @@ class TBufferedTransport extends TTransport {
   final List<int> _writeBuffer = [];
   Iterator<int> _readIterator;
 
-  Uint8List _consumeWriteBuffer() {
+  Uint8List consumeWriteBuffer() {
     Uint8List buffer = new Uint8List.fromList(_writeBuffer);
     _writeBuffer.clear();
     return buffer;
   }
 
-  void _setReadBuffer(Uint8List readBuffer) {
+  void setReadBuffer(Uint8List readBuffer) {
```

Comment:
I don't think you need to change the visibility of this method here and in all of the subclasses.